src/Documentation/Advanced-Concepts/Heterogeneous-Silos.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Heterogeneous silos
+---
+
+# Heterogeneous silos
+On a given cluster, silos can support a different set of grain types:
```

Comment:
I can see it's included manually in the HTML, but I don't see the image included *here* in the source. Am I mistaken?